src/website.cpp

```diff
@@ -250,6 +250,12 @@ std::vector<gameItem> Website::getGames()
     return games;
 }
 
+static std::string getCurlInfoString(CURL* handle, CURLINFO info)
+{
+    char* str;
+    return (curl_easy_getinfo(handle, info, &str) == CURLE_OK) ? str : "";
+}
+
```

Comment:
This helper function should be added to Util namespace and not be a static member of Website class

Reply:
Oops, it actually isn't part of the website class but just a static function.
Still it should probably be added to Util namespace like the other curl helper functions.

Reply:
I thought about that and put it near the `Website::Login` function since it is only used there. But the Util class also makes sense. I’ll update the PR.